### ISSUE TITLE
Document how to include / exclude namespace files and layout of work dir files in general

### ISSUE THREAD

**Issue by aku** [kestra-io/docs#1548]
### Description

Context: https://kestra-io.slack.com/archives/C03FQKXRK3K/p1720191206146919

Also, there is some confusion on when you actually need to inject files via namespaceFiles option. See this thread for additional context: https://kestra-io.slack.com/archives/C03FQKXRK3K/p1720619896802269

**Comment by aku** [comment]
@anna-geller Hi Anna! I've noticed that the task is closed but I can not find any info on include/exclude options of the namespaceFiles property in the commit ( Injection of all namespace files possesses security threats and potential issues. Could you please explain how to inject only certain file/directory for a task? I've tried various syntax with no luck

**Comment by anna-geller** [comment]
include/exclude is added in the plugin docs. you can either list only specific files or directories as a list or specify RegEx expressions. I agree it would be nice to add 1-2 examples, can you contribute a PR to the plugin-scripts + docs repo? 


copy-pasta:

### **`exclude`**

- **Type:**
    
    array
    
- **SubType:**
    
    string
    
- **Dynamic:** ❌
- **Required:** ❌

**A list of filters to exclude matching glob patterns. This allows you to exclude a subset of the [[Namespace Files](https://kestra.io/docs/developer-guide/namespace-files)](https://kestra.io/docs/developer-guide/namespace-files) from being downloaded at runtime. You can combine this property together with `include` to only inject a subset of files that you need into the task’s working directory.**

### **`include`**

- **Type:**
    
    array
    
- **SubType:**
    
    string
    
- **Dynamic:** ❌
- **Required:** ❌

**A list of filters to include only matching glob patterns. This allows you to only load a subset of the [[Namespace Files](https://kestra.io/docs/developer-guide/namespace-files)](https://kestra.io/docs/developer-guide/namespace-files) into the working directory.**

**Comment by aku** [comment]
@anna-geller I can contribute bud I did not figure out how to use it properly ( I've tried several paths formats but failed to include/include necessary files. I was going to check plugin source code to figure out how it works but didn't have a time to do so

**Comment by anna-geller** [comment]
as described above e.g. 

```
namespaceFiles:
  include:
    - mydir1/
    - another_dir/
```

**Comment by aku** [comment]
Hmm.. Let me try again. Maybe I missed trailing backslash or something. Thank you!
